src/plugins/discover/public/application/components/sidebar/discover_sidebar.tsx

```diff
@@ -312,6 +313,7 @@ const FieldList = ({
         size="xs"
         className="dscSideBar_fieldGroup"
         aria-label={title}
+        isLoading={selectedIndexPattern.fieldsLoading ?? false}
```

Comment:
```suggestion
        isLoading={selectedIndexPattern.fieldsLoading}
```
or coerce to boolean `!!selectedIndexPattern.fieldsLoading`?

Reply:
Coerce makes sense: [53198c4](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8609/commits/53198c40ecdcea449979f3eb51eee7782300cb54)